cmd/secret/cmd/root.go

```diff
@@ -21,11 +21,21 @@ import (
 	"github.com/go-itools-internship/go-secret/pkg/io/storage"
 	"github.com/go-itools-internship/go-secret/pkg/provider"
 	"github.com/spf13/cobra"
+	"go.uber.org/zap"
 )
 
 type root struct {
 	options options
 	cmd     *cobra.Command
+	logger  *zap.SugaredLogger
+}
+
+type chiLogger struct {
+	sugar *zap.SugaredLogger
+}
+
+func (c *chiLogger) Print(v ...interface{}) {
+	c.sugar.Info(v)
```

Comment:
Take more attention what is the signature of `c.sugar.Info` method. Pass arguments as required by the signature.